scripts/pipeline.lua

```diff
@@ -1,21 +1,27 @@
 local pipelineDepth = 1
 local counter = 0
 local maxRequests = -1
+local method = "GET"
```

Comment:
I think we should create a separate LUA script for different HTTP methods, rather than a single LUA script for both pipelining and different methods.  We currently don't have any scenarios which need both pipelining and a different HTTP method, and I don't forsee a need for this any time soon.  Keeping these scripts separate is simpler and prevents invalidating the previous results with pipeline.lua or no script.